content/stripe-cancel-survey.js

```diff
@@ -0,0 +1,11 @@
+const md = require('./template')
+
+module.exports = () => md`
+Hey there 👋,
+
+we noticed that you canceled your Greenkeeper subscription.
+To be able to improve our product and services in the future, we would be happy if you could answer a few quick questions from the cancellation survey below.
+http://bit.ly/2zs3ouq
```

Comment:
not sure where to better put this, but in the form, we have  headline “We are sorry to see you leave Greenkeeper” -> this is essentially guilt tripping the folks that potentially hate Greenkeeper, which will not help with conversion, how about something friendlier, like “thank you for trying greenkeeper”

Reply:
> Please tell us your reason for cancelling Greenkeeper

Could be friendlier too: “do you mind telling us why you cancelled greenkeeper?”

Reply:
> Any feedback on how we can improve our service to have you come back? What did you like? What did you not like

Maybe split this up into two fields, nudges people to think about what they liked. and even if they write out “nothing”, that’s better feedback than only listing stuff they don’t like.

Reply:
> Do you plan on cancelling the service altogether? Or do you plan on switching to another provider?

Add text field so they can mention the other provider

Reply:
> Do you have any comments about your customer experience that you think we should be aware of? If so, please explain in the text box below

the “please explain” is a bit harsh, maybe rephrase?

Reply:
>Please tell us your reason for cancelling Greenkeeper
I don't think this is a harsh question at all. Especially, if people read the discription 
>To improve our services in the future  we are interested in learning more about why you decided to cancel your subscription. Your responses will be kept confidential and will not be used for any purpose other than research conducted by our company.

How about: We'd love to hear your reason for cancelling Greenkeeper

Reply:
I made a few changes within the form: https://docs.google.com/forms/d/e/1FAIpQLSf8I467XybepAdXzLnjbUvYRc35wzrN9zrv_06ya0XK2HqjAA/viewform

Reply:
Heya @IrinaJay, I have a couple more comments for the form:
1. Double space in the first line between `future` and `we`
2. `Is there anything, we could do` - No comma
3. In the second-to-last question, lower case `Completely` and `Another Provider`
4. Last question, maybe do `We’d love to hear your opinions, both good & bad.`
5. Add a `Thank you for taking the time to help us out!` just before the submit button

Reply:
THX for the feedback @espy 
I updated the form accordingly.